src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1157,16 +1323,30 @@ function tryRemoveElement(elem) {
         /**
          * @private
          */
-        renderSubtitlesWithCustomElement(videoElement, track, item) {
+        renderSubtitlesWithCustomElement(videoElement, track, item, targetTextTrackIndex) {
             this.fetchSubtitles(track, item).then((data) => {
-                if (!this.#videoSubtitlesElem) {
-                    const subtitlesContainer = document.createElement('div');
-                    subtitlesContainer.classList.add('videoSubtitles');
-                    subtitlesContainer.innerHTML = '<div class="videoSubtitlesInner"></div>';
+                if (!this.#videoSubtitlesElem && !this.isSecondaryTrack(targetTextTrackIndex)) {
+                    let subtitlesContainer = document.querySelector('.videoSubtitles');
+                    if (!subtitlesContainer) {
+                        subtitlesContainer = document.createElement('div');
+                        subtitlesContainer.classList.add('videoSubtitles');
+                    }
+                    const subtitlesElement = document.createElement('div');
+                    subtitlesElement.classList.add('videoSubtitlesInner');
+                    subtitlesContainer.appendChild(subtitlesElement);
                     this.#videoSubtitlesElem = subtitlesContainer.querySelector('.videoSubtitlesInner');
                     this.setSubtitleAppearance(subtitlesContainer, this.#videoSubtitlesElem);
                     videoElement.parentNode.appendChild(subtitlesContainer);
                     this.#currentTrackEvents = data.TrackEvents;
+                } else if (!this.#videoSecondarySubtitlesElem && this.isSecondaryTrack(targetTextTrackIndex)) {
+                    const subtitlesContainer = document.querySelector('.videoSubtitles');
+                    if (!subtitlesContainer) return;
+                    const secondarySubtitlesElement = document.createElement('div');
+                    secondarySubtitlesElement.classList.add('videoSecondarySubtitlesInner');
+                    subtitlesContainer.prepend(secondarySubtitlesElement);
+                    this.#videoSecondarySubtitlesElem = secondarySubtitlesElement;
+                    this.setSubtitleAppearance(subtitlesContainer, this.#videoSecondarySubtitlesElem);
```

Comment:
`setSubtitleAppearance` requires some fix.
Force using of custom element and try to change `Vertical Position` in the Subtitles Settings to a positive value (3).
In case of positive position `videoSecondarySubtitlesInner` should go after `videoSubtitlesInner`.

Reply:
This is a really great point. I had not at all considered the vertical position setting. I will swap ordering depending on a positive or negative value.